### PR TITLE
Adding security hardening guide

### PR DESCRIPTION
**Pull request by asettle**
Supersedes #561

### CODE REVIEW

**Comment by asettle** [comment]
[book-storage-security_color_en.pdf](https://github.com/SUSE/doc-ses/files/5224628/book-storage-security_color_en.pdf)

File: xml/hardening_measures_confidentiality.xml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  <para>
+    Confidentiality is a common requirement. There are diffent ways of ensuring
+    data stays confidential at different times of its lifecycle.
+  </para>
+
+ <sect1 xml:id="data-at-rest">
+   <title>Data at rest</title>
+   <para>
+    The data stored in OSDs is not encrypted by default. If you have high
+    confidentiality requirements, we recommendethat you encrypt the storage
```
Comment (ajaeger) [inline xml/hardening_measures_confidentiality.xml:18]: extra e, should be: "recommend that"

**Review by ajaeger — CHANGES_REQUESTED** [review]
LGTM as first version, please fix the typo.

File: xml/hardening_measures_confidentiality.xml (hunk 1 above)
Comment (asettle) [inline xml/hardening_measures_confidentiality.xml:18]:
```suggestion
    confidentiality requirements, we recommend that you encrypt the storage

**Comment by asettle** [comment]
> LGTM as first version, please fix the typo.

Thanks. Fixed. Approve when you're ready :)

**Review by ajaeger — APPROVED** [review]
Thanks!

**Pull request merged**